src/main/java/org/folio/gobi/LookupService.java

```diff
@@ -251,11 +248,7 @@ public CompletableFuture<String> lookupAcquisitionUnitIdsByName(String data) {
 
   public CompletableFuture<Object> lookupAcquisitionUnitIdsByAccount(String data) {
     logger.debug("lookupAcquisitionUnitIdsByAccount:: Trying to look up acquisitionUnitIds by accountNumber: {}", data);
-    return lookupOrganization(ORGANIZATION_NAME).thenApply(org -> org.getAccounts().stream()
-      .filter(acc -> HelperUtils.normalizeSubAccout(acc.getAccountNo()).equals(HelperUtils.normalizeSubAccout(data)))
-      .findFirst()
-      .map(Account::getAcqUnitIds)
-      .orElse(null));
+    return CompletableFuture.completedFuture(data);
```

Comment:
There is no lookup happening here. So it looks like this proxying makes no sense

Reply:
Organization can have similar accountNo , 
so if we try to lookup by account No , we will endup getting multiple organizations , and we have to filter the organization by Vendor code . 

anyway Vendor is a mendatory field 
so i am thinking  instead of getting multiple organizations we can directly lookup organization and match account No. 

Kindly suggest if any better way . Thanks

Reply:
@SinghAdes @grabsefx I don't understand why we can't remove the lookUp method?